src/constants/skills.js

```diff
@@ -425,7 +427,8 @@ const icons = {
     "https://raw.githubusercontent.com/AliasIO/wappalyzer/master/src/drivers/webextension/images/icons/VuePress.svg",
   unity: "https://www.vectorlogo.zone/logos/unity3d/unity3d-icon.svg",
   unreal:
-    "https://raw.githubusercontent.com/kenangundogan/fontisto/036b7eca71aab1bef8e6a0518f7329f13ed62f6b/icons/svg/brand/unreal-engine.svg",
+    "https://icon-library.com/images/unreal-engine-icon/unreal-engine-icon-25.jpg",
+    godot:"https://upload.wikimedia.org/wikipedia/commons/6/6a/Godot_icon.svg",
```

Comment:
Why not use the one from godot https://raw.githubusercontent.com/godotengine/godot/master/icon.svg

Reply:
Okay implementing this asap

Reply:
This is resolved I guess!